pkg/apis/lakom/types.go

```diff
@@ -0,0 +1,37 @@
+// SPDX-FileCopyrightText: 2022 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+
+package lakom
+
+import (
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+
+// ScopeType determines the namespaces and labels that will be monitored by lakom webhooks
+type ScopeType string
+
+const (
+	// KubeSystem denotes the `kube-system` namespace
+	KubeSystem ScopeType = "kubeSystem"
```

Comment:
Use pascal case for the enumerated values, ref: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#constants

```suggestion
	KubeSystem ScopeType = "KubeSystem"
```